.github/CODEOWNERS

```diff
@@ -2099,27 +2101,32 @@ x-pack/platform/plugins/shared/ml/server/models/data_recognizer/modules/security
 #CC# /src/platform/plugins/shared/newsfeed @elastic/kibana-core
 #CC# /x-pack/plugins/global_search_providers/ @elastic/kibana-core
 
-# AppEx AI Infra
-/x-pack/platform/plugins/shared/inference @elastic/appex-ai-infra
-/x-pack/platform/test/functional_gen_ai/inference @elastic/appex-ai-infra
-
-# AppEx Platform Services Security
-/x-pack/platform/test/serverless/api_integration/test_suites/security_response_headers.ts  @elastic/kibana-security
-/x-pack/platform/test/api_integration/apis/es  @elastic/kibana-security
-/x-pack/platform/test/api_integration/apis/features  @elastic/kibana-security
+# Exclude OAS error baseline file from @elastic/kibana-core ownership
+!src/platform/packages/private/kbn-validate-oas/oas_error_baseline.json
+# Exclude usage collection schema from @elastic/kibana-core ownership
+!src/platform/plugins/private/kibana_usage_collection/server/collectors/application_usage/schema.ts
 
 # Kibana Telemetry
 /.telemetryrc.json @elastic/kibana-core
 /x-pack/.telemetryrc.json @elastic/kibana-core
-/src/platform/plugins/shared/telemetry/schema/ @elastic/kibana-core @elastic/kibana-telemetry
-/x-pack/platform/plugins/private/telemetry_collection_xpack/schema/ @elastic/kibana-core @elastic/kibana-telemetry
+/src/platform/plugins/shared/telemetry/schema/ @elastic/kibana-telemetry
+/x-pack/platform/plugins/private/telemetry_collection_xpack/schema/ @elastic/kibana-telemetry
 x-pack/platform/plugins/private/cloud_integrations/cloud_full_story/server/config.ts @elastic/kibana-core @shahinakmal
 
 # Kibana Localization
 /src/dev/i18n_tools/  @elastic/kibana-localization @elastic/kibana-core
 /src/core/public/i18n/  @elastic/kibana-localization @elastic/kibana-core
 #CC# /x-pack/platform/plugins/private/translations/ @elastic/kibana-localization @elastic/kibana-core
 
+# AppEx AI Infra
+/x-pack/platform/plugins/shared/inference @elastic/appex-ai-infra
+/x-pack/platform/test/functional_gen_ai/inference @elastic/appex-ai-infra
+
+# AppEx Platform Services Security
+/x-pack/platform/test/serverless/api_integration/test_suites/security_response_headers.ts  @elastic/kibana-security
+/x-pack/platform/test/api_integration/apis/es  @elastic/kibana-security
+/x-pack/platform/test/api_integration/apis/features  @elastic/kibana-security
+
```

Comment:
no changes, just moved around to keep core, telemetry, localization closer together